scripts.js

```diff
@@ -24,8 +26,8 @@ const exec = (cmd, env) => {
 
 const prepare = () => {
   // Copy secrets.sample.json to secrets.json if needed
-  const secretsPath = path.join(__dirname, 'secrets.json');
-  const secretsSamplePath = path.join(__dirname, 'secrets.sample.json');
+  const secretsPath = new URL('./secrets.json', import.meta.url);
```

Comment:
Will this really work without the `fileURLToPath` bit? Does `fs.existsSync` accept a file:// URL?

Reply:
It seems that file URLs are indeed supported by `fs`, see https://nodejs.org/dist/latest-v14.x/docs/api/fs.html#fs_url_object_support.  From what I've also researched on that page, we can also use relative paths, which will resolve relative to `process.cwd()`, which we only change when preparing Puppeteer.